zcash_client_backend/src/constants/regtest.rs

```diff
@@ -0,0 +1,39 @@
+//! # Regtest constants
+//!
+//! `regtest` is a `zcashd`-specific environment used for local testing. They mostly reuse
+//! the testnet constants.
+//! These constants are defined in [the `zcashd` codebase].
+//! [the `zcashd` codebase]: https://github.com/zcash/zcash/blob/128d863fb8be39ee294fda397c1ce3ba3b889cb2/src/chainparams.cpp#L482-L496
+
+/// The regtest cointype reuses the testnet cointype
+pub const COIN_TYPE: u32 = 1;
+
+/// The HRP for a Bech32-encoded regtest [`ExtendedSpendingKey`].
+///
+/// It is defined in [the `zcashd` codebase here].
```

Comment:
```suggestion
/// It is defined in [the `zcashd` codebase].
```
Or add `here` to the link definition. Either way, they need to match for the documentation to correctly build.